src/App.js

```diff
@@ -1,9 +1,116 @@
-function App() {
+import React, { useState, useEffect } from "react";
+import { 
+  AppContainer, Main, TodoContainer, MainText, Line, TodosBody,
+  TodoHeader, TodoList, TodoItem, TodoInput,
+  Input, Button, RemoveButton 
+} from './styles';
```

Comment:
style.js에서 
```
export const S = {
  AppContainer, Main, ...
};
```
과 같이 씀으로써 export도 일일이 쓰지 않고

```suggestion
import { S } from './styles';
```
import문은 파일 자체를 지정하여 아래쪽 return 부에서 컴포넌트를 쓸 때 `<S.MainText></S.MainText>` 와 같이 쓰면 더 간단할 것 같아요!

Reply:
오.. 지원님이 말씀해주신 방법 정말 간결하고 좋네요